bundles/http/Citrix/Microsoft-Outlook/metadata.json

```
@@ -1,39 +1,40 @@
-{
-  "type" : "HTTP",
-  "vendor" : "Citrix",
-  "id" : "4b449ffb-d1fd-4244-b68c-7dac9c3b80cf",
-  "trackingUuid" : "6161ed59-c301-4e8a-a517-5acfda89cf0d",
-  "title" : "Microsoft Outlook",
-  "description" : "Deploy the Microsoft Outlook integration to schedule Event and Office Hours, edit Event and Office Hours, and receive a notification before an hour the event start time.",
-  "iconUrl" : "https://iws-prod-global-customicons-cdn-endpoint.azureedge.net/image/ctMmybALGSmNfPLyVbZY1LKAtVBAA0Ni.png",
-  "masVersion" : "1.52.1.d19ef4fc0b67e48447629f7a9f403d484d2bb55f",
-  "created" : "2021-01-11T09:07:42",
-  "categories" : ["WEB_SERVICES"],
-  "supportsOAuthForActions" : true,
-  "i18nLanguages" : [ "de", "en", "es", "fr", "it", "ja", "nl", "pt-BR", "zh-CN" ],
-  "apps" : [ {
-    "uuid" : "1a1314b6-c9e3-4a6e-9362-a3d1f6ef2664",
-    "trackingUuid" : "541cd3f0-0034-43f8-85f1-cf4f0bc50adb",
-    "title" : "My Office Hours",
-    "numberOfEvents" : 0,
-    "action" : true
-  }, {
-    "uuid" : "85e7647b-7c23-453a-b4b6-80ce16b7b273",
-    "trackingUuid" : "e6210fc5-5c7c-474e-99e1-4107955ac863",
-    "title" : "Create Event",
-    "numberOfEvents" : 0,
-    "action" : true
-  }, {
-    "uuid" : "8968a350-e147-46d3-a876-5a5486073c8a",
-    "trackingUuid" : "7c0f4985-26ff-471a-94ea-c042adda7aa3",
-    "title" : "My Calendar",
-    "numberOfEvents" : 1,
-    "action" : true
-  } ],
-  "vaResolvers" : [ ],
-  "tags" : [{
-   "tag" : "helpUrl",
-   "value" : "https://docs.citrix.com/en-us/citrix-microapps/set-up-template-integrations/integrate-ms-outlook.html"
-   }],
-  "hideAddButton" : false
-}
+{
+  "type" : "HTTP",
+  "vendor" : "Citrix",
+  "id" : "82ce88e5-306a-4501-80ca-55b3147b3b5e",
+  "trackingUuid" : "39206030-5dee-47ea-a673-68c8317f554d",
+  "title" : "MS Outlook Second Set",
+  "iconUrl" : "exported/HTTP.9b81384ab8a11658b635e7542ff6416c.svg",
+  "masVersion" : "1.136.0.843ffba479fd46f0a40992f38bd38a0ec4471dad",
+  "categories" : [ ],
+  "created" : "2021-12-10T06:45:59",
+  "supportsOAuthForActions" : true,
+  "i18nLanguages" : [ "de", "en", "es", "fr", "it", "ja", "nl", "pt-BR", "zh-CN" ],
+  "apps" : [ {
+    "uuid" : "0fa643c5-eda6-4711-b248-635f1abd317c",
+    "trackingUuid" : "e6210fc5-5c7c-474e-99e1-4107955ac863",
+    "title" : "Create Event",
+    "numberOfEvents" : 0,
+    "action" : true
+  }, {
+    "uuid" : "95ff6d74-6f3f-46cb-8942-97bff04b0949",
+    "trackingUuid" : "541cd3f0-0034-43f8-85f1-cf4f0bc50adb",
+    "title" : "My Office Hours",
+    "numberOfEvents" : 0,
+    "action" : true
+  }, {
+    "uuid" : "a5ae3f3b-4b9f-4534-8e13-be39b27ac2ca",
+    "trackingUuid" : "7c0f4985-26ff-471a-94ea-c042adda7aa3",
+    "title" : "My Calendar",
+    "numberOfEvents" : 1,
+    "action" : true
+  } ],
+  "tags" : [ ],
```

`helpUrl` is missing in the `tags` array. Update the same as previous integration.
```suggestion
  "tags" : [{
   "tag" : "helpUrl",
   "value" : "https://docs.citrix.com/en-us/citrix-microapps/set-up-template-integrations/integrate-ms-outlook.html"
   }],
```

---

Please update this changes.